### ISSUE TITLE
Please add autocomplete="off"

### ISSUE THREAD

**Issue by IliyanTatarov** [mattzollinhofer/vue-typeahead-bootstrap#107]
Please add autocomplete="off" to the search input.

**Comment by MaBre01** [comment]
Hi!
I had a similar problem.

The problem result from the browser which display suggestions on the suggestions container of the `vue-typeahead-bootstrap` component (but it does not appear on all my components).

I fixed it with adding `autocomplete="off"` props on the input with VueJS refs in the mounted function.

**Comment by darrinmn9** [comment]
I am also experiencing this problem. Browser based autocomplete suggestions are being shown on top of the suggestions items list provided by this component, leading to ugly UI and many times being unable to see the options suggested by this component (because chrome/browser is completely overlapping it).

**Comment by darrinmn9** [comment]
>  A major priority of this project is to be as accessible as possible, so any thoughts you have on these changes and how they relate to the attributes I mentioned would be appreciated.

https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/autocomplete

I can do some research on this, but I don't believe `autocomplete="off"` negatively affects a11y. The feature is more about the allowing the user's browser to suggest hints (in a dropdown) of what the user could be trying to type (based on w/e logic the browser chooses). Here is a very strict, WAI-ARIA compliant library that suggests that option as well. https://bootstrap-vue.org/docs/components/form-input#datalist-support

**Comment by mattzollinhofer** [comment]
Yep, thanks for looking into the accessible side a bit, this sounds good, let's set it to `"off"`
